### ISSUE TITLE
Module is non-functional

### ISSUE THREAD

**Issue by Baddiemcnoskill** [jopeek/fvtt-loot-sheet-npc-5e#349]
On Foundry Version 9, build 269, 5th edition version 1.6.1

Trying to drag items back and forth, using the Trade, Take All, or Split Currency buttons does nothing.

**Comment by DanielBoettner** [comment]
What does the error console of your browser say?

When Split Currency doesn't work I'd like more info on what the situation was (GM, 2/3/4 players, all eligible, ...).
'Trade' and 'Take All' are referencing the stage. So whatever you clicked to be on the stage will be handled when you click a button.

Also the Trade view currently is clicking not dragging.

**Comment by Baddiemcnoskill** [comment]
Well, I'm terribly sorry. Yesterday, nothing was working at all. Every time I clicked on anything, it would produce no result. I was seeing a lot of other posts on the Foundry discord saying it was broken for other people too.

But today, I went back to get you your error logs and it functions now. I changed _literally nothing_ and it works now. Despite a lot of restarting and closing and reopening of Foundry and modules and whatnot yesterday.

A real head scratcher for sure.
